content/docs/command-reference/update.md

```diff
@@ -49,6 +49,20 @@ $ dvc update --rev master
   directory and its subdirectories for import stage `.dvc` files to inspect. If
   there are no directories among the targets, this option is ignored.
 
+- `--to-remote` - update the import `.dvc` file and
+  [transfer](/doc/command-reference/import-url#example-import-straight-to-the-remote)
+  the new data directly to remote storage (the default one unless `-r` is used).
+  No changes are done in the <abbr>workspace</abbr>. Use `dvc pull` to get the
+  data locally.
```

Comment:
We forgot to mention that this only works for `import-url` imports @isidentical — per iterative/dvc/pull/5475

Reply:
p.s. can you `update --to-remote` data `import-url`ed without `--to-remote`? I assume there's no restriction to do so but just curious.

Reply:
> p.s. can you update --to-remote data import-urled without --to-remote

yes